mrtt-ui/src/components/EcologicalStatusAndOutcomes/EcologicalStatusAndOutcomesForm.js

```diff
@@ -64,35 +68,62 @@ const EcologicalStatusAndOutcomesForm = () => {
   } = reactHookFormInstance
 
   const { siteId } = useParams()
-  const apiAnswersUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/registration_intervention_answers`
+  const interventionAnswersUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/intervention_answers`
+  const monitoringFormsUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/monitoring_answers`
+  const monitoringFormSingularUrl = `${monitoringFormsUrl}/${monitoringFormId}`
   const [isSubmitting, setIsSubmitting] = useState(false)
   const [isSubmitError, setIsSubmitError] = useState(false)
-  const [isLoading, setIsLoading] = useState(false)
+  const [isMainFormDataLoading, setIsMainFormDataLoading] = useState(false)
+  const [areBiophysicalInterventionsLoading, setAreBiophysicalInterventionsLoading] =
+    useState(false)
   const [biophysicalInterventions, setBiophysicalInterventions] = useState([])
 
-  const loadServerData = useCallback((serverResponse) => {
-    const biophysicalInterventionsInitialVal = getBiophysicalInterventions(serverResponse)
-
-    setBiophysicalInterventions(biophysicalInterventionsInitialVal.selectedValues)
-  }, [])
+  useEffect(
```

Comment:
made this just a normal effect rather than a callback since it doesnt depend on the server response from loading the monitoring form

Reply:
aren't we still pulling the answer set from the server? is this due to recent backend changes? @mmnoo

Reply:
no this is just the callback for the custom hook. For the other forms, we pass the loaded server data to a callback function. Here we arent loading stuff from the monitoring_whatever endpoint. We need to hit a different endpoint

Reply:
god, sorry, communication is harder than usual for me today. The old hook had the call back function it provided with data form `registration_answers` to parse out info from other parts of the form not parsed by the old custom hook. In this form, we need to load data from the `monitoring_x` endpoint, and also another separate endpoint

Reply:
so this is just for monitoring forms. The other forms stay the same